src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -56,37 +50,21 @@ const SetAclPermsDocContainerForm = () => {
     try {
       await setDocContainerAclPermission(session, permissions, podUrl, webIdToSetPermsTo);
 
-      runNotification(
-        `${permissions.read ? 'Give' : 'Revoke'} permission to ${
-          permissionState.podUrlToSetPermissionsTo
-        } for Documents Container.`,
-        5,
-        state,
-        dispatch
-      );
       addNotification(
         'success',
         `${permissions.read ? 'Gave' : 'Revoked'} permission to ${
```

Comment:
I think the "to" should also be a ternary based on `permissions.read`.  I'd prefer it to be `${permissions.read ? 'to' : 'from'}` as just "to" doesn't really work for `Revoked permissions to {person} for {doctype}`.

Reply:
Just like the other one